app/components/date/custom-date-picker.tsx

```diff
@@ -0,0 +1,106 @@
+import DatePicker from 'react-datepicker'
```

Comment:
You see a warning but this is depending on DatePicker library...
<img width="635" alt="Screenshot 2024-01-30 at 12 14 26 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/175616a1-9e47-47c9-9af2-e7617e61d322">